srcpkgs/gnunet/INSTALL.msg

```diff
@@ -0,0 +1 @@
+Please modify your /etc/gnunet/gnunet.conf and start gnunet.
```

Comment:
This message doesn't look especially useful. What will happen when default config is used?

Reply:
Oh well it can be used but very basic. Most configs I‘ve found are quite complex but very specific. Should I remove the msg?

Reply:
Yes, I think people do know that programs can be configured.